tests/suite_test.go

```diff
@@ -209,3 +288,11 @@ func (suite *IntegrationTestSuite) TearDownSuite() {
 func TestIntegrationTestSuite(t *testing.T) {
 	suite.Run(t, new(IntegrationTestSuite))
 }
+
+func (t *customTransport) RoundTrip(req *http.Request) (*http.Response, error) {
+	// Add the Authorization header globally
+	req.Header.Set("Authorization", fmt.Sprintf("Bearer %s", t.Token))
```

Comment:
Just to be sure... Did you replace the standard HTTP client because you needed to set this header?

Reply:
The header for authentication on OpenShift and the Transport. Now, we are ignoring the cert but we are going to implement it soon

```
		suite.Clientmgr.httpClient.Transport = &customTransport{
			Transport: &http.Transport{
				TLSClientConfig: &tls.Config{InsecureSkipVerify: true},
			},
			Token: cfg.BearerToken,
		}
```